### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by gfoligna** [mivek/MetarParser#385] [Ideas]
As a native Spanish speaker I could help translating the website to get a broader audience, and to pay for the `strict mode` request in the other idea I've created. 🙂

**Comment by mivek** [comment]
Hello @gfoligna 

Thank you for your interest in this project. If you are willing to contribute in the translation you can use the [crowdin link](https://crwd.in/metarParser). Once the translation is complete, I will add the translation to the project.
